CMakeLists.txt

```diff
@@ -186,6 +184,8 @@ if(NOT EMSCRIPTEN)
     list(APPEND XEUS_CPP_SRC
         src/xmagics/xassist.hpp
         src/xmagics/xassist.cpp
+        src/xmagics/os.hpp
```

Comment:
Do we need to list the local `hpp` files?

Reply:
Guessing we can do without them. Trying it out.

Reply:
The tests pass, so I think we can do without them